lib/note-actions/index.tsx

```diff
@@ -93,8 +93,8 @@ export class NoteActions extends Component<Props> {
               <span className="note-actions-item-control">
                 <CheckboxControl
                   id="note-actions-pin-checkbox"
-                  className="theme-color-border"
                   checked={isPinned}
+                  className="standard"
```

Comment:
I wasn't sure about the best way to set the standard styles, wdyt? would you rather have a prop for picking it?

Reply:
Thanks so much for picking this up. This makes good sense to move it into the component. 
I wasn't sure the best way to do this either. It looks like we're going to bring in the WordPress Components to Simplenote Electron/Web at some point soon and it seems they use props to set options like this on components. Maybe we should do that also to be consistent.

Reply:
Ok, makes sense, I'm going to move it to a prop then 👍 .

Reply:
I applied this change in the following [commit](https://github.com/Automattic/simplenote-electron/pull/2835/commits/1e09b59c135b11ade2ee96abb081ab3f9ac53d16).